src/encoded/authentication.py

```diff
@@ -407,16 +433,15 @@ def login(context, request):
 @debug_log
 def logout(context, request):
     """
-    This endpoint proxies a request to Auth0 for it to remove its session cookies.
-    See https://auth0.com/docs/api/authentication#enterprise-saml-and-others-
-
-    The Auth0 endpoint is meant to be navigated to by end-user as part of SSO logout (?)
-    So this endpoint may not be needed at moment. Kept for reference.
-
-    The front-end handles logging out by discarding the locally-held JWT from
-    browser cookies and re-requesting the current 4DN URL.
+    This endpoint is called by the front-end upon executing a logout. It will delete
+    the session token passed to it from Redis storage, which has the effect of logging
+    out the user internally. If this isn't done there is no harm as the session token
+    will expire after 3 hours anyway. 
+
+    We do not send the logout signal to Auth0 at this time, since we handle it on our end.
+    It may be undesirable to send this signal as it will log the user out of all sessions
+    tied to that account.
```

Comment:
Yeah, that (undesirable - log the user out of all sessions tied to that account) sounds right.